<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQTPasswordDialog</class>
 <widget class="QQTDialog" name="QQTPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>30</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>WIFI:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ssid_password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QQTPushButton" name="pushButton_ssid_connect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ssid_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>31</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SSID NAME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QQTPushButton</class>
   <extends>QPushButton</extends>
   <header>qqtpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>QQTDialog</class>
   <extends>QWidget</extends>
   <header>qqtdialog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
